RecoLocalTracker/SubCollectionProducers/src/SeedClusterRemoverPhase2.cc

```diff
@@ -61,7 +61,9 @@ using namespace std;
 using namespace edm;
 
 SeedClusterRemoverPhase2::SeedClusterRemoverPhase2(const ParameterSet &iConfig)
-    : doOuterTracker_(iConfig.existsAs<bool>("doOuterTracker") ? iConfig.getParameter<bool>("doOuterTracker") : true),
+    : tTrackerGeom_(
+          esConsumes<TrackerGeometry, TrackerDigiGeometryRecord>(edm::ESInputTag("", ""))),  //is it correct to use "" ?
```

Comment:
the comment looks like a copy of that from the non-Phase2 variant.
 is it reasonable to expect that the comment is not necessary anymore?

Reply:
see above.